ISTC.js

```diff
@@ -0,0 +1,199 @@
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	if ( getSearchResults(doc, true, url) ) {
+		if (url.match('_search?')){
+			return "multiple";
+		}
+		else if (url.match(/i[a-z]\d{8}/)){
+			return 'book';
+		}
+	}
+}
+
+
+function getSearchResults(doc, checkOnly, url) {
+	var items = { };
+	var found = false;
+	if (url.match('_search?')){
+		var rows = ZU.xpath(doc, '//p[contains(@class, "ample-shortlist-item-entry")]/a[contains(@href, "/istc/i")]' );
+		for (i=0; i<rows.length; i++) {
+			var title = ZU.trimInternal(rows[i].textContent);
+			var href = rows[i].href + '?format=json';
+			if (!href || !title) continue;
+			if (checkOnly) return true;
+			found = true;
+			items[href] = title;
+		}
+	}
+	else {
+		var title = ZU.trimInternal( ZU.xpath(doc, '//div[contains(@class, "ample-record")]/h3' )[0].textContent );
+		if (!title) return false;
+		if (checkOnly) return true;
+		items[url] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if ( detectWeb(doc, url) == "multiple" ) {
+		Zotero.selectItems( getSearchResults(doc, false, url), function (items) {
+			if (!items) {
+				return true;
+			}
+			var books = [];
+			for (var i in items) {
+				books.push(i);
+			}
+		ZU.doGet(Object.keys(items), scrape);
+		} );
+	} 
+	else if ( detectWeb(doc, url) == "book" ){
+		ZU.doGet( url + '?format=json', scrape );
+	}
+}
+
+
+function scrape( response, obj, url ) {
+		var jsonObject = JSON.parse( response );
+		var item = new Zotero.Item();
+		var name =  jsonObject.data.author;
+		var cleanNameObj = Zotero.Utilities.cleanAuthor(name, "author", true);
+		item.creators[0] = cleanNameObj;
+		item.title = jsonObject.data.title;
+		item.url = url.replace('?format=json', '');
+		item.place = jsonObject.data.imprint[0].imprint_place;
+		item.publisher = jsonObject.data.imprint[0].imprint_name;
+		item.date = jsonObject.data.imprint[0].imprint_date;
+		item.notes = [ jsonObject.data.imprint[0].geo_info.geonames_id || '' ] ;
+		item.callNumber = 'ISTC' + jsonObject._id;
+		item.extra = jsonObject.data.notes[0];
+		item.language = jsonObject.data.language_of_item;
+		item.seeAlso = [ jsonObject.data.references[0].reference_name + ' ' + jsonObject.data.references[0].reference_location_in_source ];

```

Comment:
I think you cannot save this information into `seeAlso` because that field is for relations to other Zotero items. What is this exactly you want to save here?